### DISCUSSION TITLE
Branding the package

### DISCUSSION THREAD

**Discussion by thomasarsouze** [umr-amap/StormR#16] [Ideas]
Hi @thomaspibanez and @BaptisteDlp 

Two things we need to think about as the release party is coming:
- think of a logo. Any idea ? Should we brainstorm about keywords of the package ? Should we ask some friends who have design skills ? Should we ask some AI ?
- Is it necessary to change the website name to get rid of the "umr-amap" ? We can ask IRD's DSI to do so if we think we do.

**Comment by thomaspibanez** [comment]
* Asking some AI might help to same time
* Happy to keep "umr-amap" in the website
